app/views/layouts/_footer.html.erb

```diff
@@ -2,7 +2,7 @@
   <div id="footer-container" class="container <%= "centered-footer" unless current_page.include?("stories-show") %>">
     <div class="inner-footer-container">
       <a href="/">Home</a> <a href="/about">About</a> <a href="/privacy">Privacy Policy</a> <a href="/terms">Terms of Use</a> <a href="/contact">Contact</a> <a href="/code-of-conduct">Code of Conduct</a>
-      DEV Community copyright 2018 🔥
+      DEV Community copyright 2016 - <%= Time.new.year %> 🔥
```

Comment:
I would just display the current year, the same way it was before

Reply:
I probably would've done well to ask first. Might not be a bad idea to consult with an IP attorney, out of an abundance of caution, assuming the organization's intent to enforce the copyright. I have found multiple attorneys who offer similar guidance on their respective blogs and reference sites: https://danashultz.com/2013/10/09/copyright-notice-with-multiple-years-legitimate/

It's up to you. Thanks for the opportunity to contribute!